frontend/src/metabase/visualizations/visualizations/AreaChart.jsx

```diff
@@ -20,6 +24,9 @@ export default class AreaChart extends LineAreaBarChart {
   static iconName = "area";
   static noun = t`area chart`;
 
+  static minSize = getMinSize(this.identifier);
```

Comment:
some vizs use its name instead of `this.identifier` even though it also has `identifier` e.g. `getMinSize("text")`.

I think we should stay consisten either to use `getMinSize(this.identifier)` or `getMinSize("vizName")` here

What do you think?